.zapierapprc

```diff
@@ -0,0 +1,4 @@
+{
+  "id": 179570,
```

Comment:
Is this sensitive info? Is it ok to have in a public repo?

Also, do we need the `inlcudeInBuild` section for LICENSE or other files?

Reply:
I think just these 2 fields are ok, can add more later as required